### PR TITLE
Fix Select2 on List Widget

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#67091](https://central.tri.be/issues/67091)

### CODE REVIEW

File: src/resources/js/events-admin.js [hunk 1]
```diff
@@ -710,16 +710,53 @@ jQuery( document ).ready( function( $ ) {
 
 });
 
+( function ( $ ) {
+	'use script';
```
Comment (barryhughes) [inline src/resources/js/events-admin.js]: `'use strict'`?

File: src/resources/js/events-admin.js [hunk 2]
```diff
@@ -710,16 +710,53 @@ jQuery( document ).ready( function( $ ) {
...
- * credits: http://www.johngadbois.com/adding-your-own-callbacks-to-wordpress-ajax-requests/
- */
-jQuery( document ).ajaxSuccess( function( e, xhr, settings ) {
-	if ( typeof settings !== 'undefined' && typeof settings.data !== 'undefined' && settings.data.search( 'action=save-widget' ) != - 1 ) {
-		jQuery( "#widgets-right .chosen" ).chosen();
-	}
-} );
+	// Track when the Widget Opens
+	$( document.body ).on( 'click.widgets-toggle', function( e ) {
+		var target = $( e.target ),
```
Comment (barryhughes) [inline src/resources/js/events-admin.js]: Add a sigil to mark these vars as likely to contain jQuery objects? ie `$target`, `$widget`, `$inside`

File: src/resources/js/events-admin.js [hunk 3]
```diff
@@ -710,16 +710,53 @@ jQuery( document ).ready( function( $ ) {
...
+		if ( ! widget.find( '.select2-container' ).hasClass( 'select2-offscreen' ) ) {
+			return;
+		}
+
+		widget.find( 'select.calendar-widget-add-filter' ).removeClass( 'select2-offscreen' ).select2();
+		calendar_toggle_all();
+	} );
+
+	// When Updated Re-Structure the Select2
+	$( document ).on( 'widget-updated', function ( e, widget ) {
```
Comment (barryhughes) [inline src/resources/js/events-admin.js]: Ditto: consider renaming `widget` to `$widget`

File: src/resources/js/events-admin.js (hunk 1 above)
Comment (bordoni) [inline src/resources/js/events-admin.js]: thanks

**Comment by bordoni** [comment]
Ready for another pair of eyes.

File: common [hunk 4]
```diff
@@ -1 +1 @@
-Subproject commit f0cd14a39ef80a2b8d57ce9fe3437b8a73c2d722
+Subproject commit ce12c5a2ffe9f6d5d1f6d916d322a72eeef658b4
```
Comment (borkweb) [inline common]: This is a really old common hash from ~18 days ago.

**Comment by borkweb** [comment]
:+1:

**Pull request merged**